src/tests/TestProviders.tsx

```diff
@@ -36,11 +39,13 @@ type Props = {
 
 function TestProviders({ mocks, children, router }: Props) {
   return (
-    <RouterContext.Provider value={{ ...mockRouter, ...router }}>
-      <MockedProvider mocks={mocks} addTypename={false}>
-        {children}
-      </MockedProvider>
-    </RouterContext.Provider>
+    <I18nextProvider i18n={i18n}>
```

Comment:
Did you try without the provider?

Reply:
Attempted multiple approaches. Some did not use a provider.